data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,244 @@ mission "Remnant: Visit to Paradise 1"
+			`	As you slow to a hover alongside the ship, Ruach appears on your viewscreen. "Well met, Captain! Our theory worked! We managed to snap the Shattered Light back into the primary weft, and are now onboard. The systems are dubious, though, so we dare not shut anything down lest we cannot restart it. We would appreciate your assistance in escorting us back to Viminal."`
+				accept
+	npc accompany
+		government "Remnant (Research)"
+		personality launching timid escort
+		planet "Far Monad"
+		ship "Modified Dromedary Wreck" "Shattered Light"
+	npc evade
+		government "Korath"
+		personality entering plunders harvests
+		system Lucina
+		fleet "Korath Ember Waste Raid" 1
+	on complete
+		payment 250000
+		clear "Remnant: Shattered Light 3b: failed"
+		set "Remnant: Shattered Light 3b: done"
+		conversation
+			`	The Shattered Light is so massive that traffic control directs you to a large valley next to the spaceport, where you discover that a section of the chasm has been covered to create a semi-enclosed area. Judging from the equipment and stacks of cargo, this area looks like it is predominantly used by bulk freight shipments and little else. Now, however, as the Shattered Light slowly moves in, a flock of Pelicans scatter in front of it looking like nothing more than a flock of plovers in front of an approaching camel.`
+			`	Down on the ground, you see Ruach and her team exit an airlock on the side of the Shattered Light and jump down to the ground. They gather around, giving hugs. "<first>!" exclaims Ruach. "We have recovered a lost piece of Remnant history. Thank you for helping. I do not know what this will mean for us in the long term, but there will be a lot of careful examination for the next few weeks."`
+			choice
+				`	"I have things to do, so just let me know if I can help."`
+					goto shatteredhelp
+				`	"So, what happened?"`
+			`	The team of Remnant are almost too excited for you to understand their gestures, but one calms down enough that their signing is comprehensible. "To be honest, we were expecting this to be the first of several trips doing detailed scans and trying out different sorts of synchronization fields. But when we saw that it was heading to the planet..." She trails off, and another one continues. "We already had the sensors running as we approached, so we jumped and triggered the fields as soon as we got a reasonable match."`
+			`	He frowns a bit. "It worked, although not perfectly. There are chunks missing, some of which seriously impair its functionality. But it is back in the primary weft, so now we have time to work on it and figure out what happened to it.`
+			choice
+				`	"I think I get the picture. Let me know if I can help."`
+					goto shatteredhelp
+				`	"What is this 'Primary Weft' you have mentioned?"`
```

Comment:
"Weft" and "Primary Weft" aren't capitalized up until you ask this question, but then it sometimes goes back to lower case after this.. Should they always be capitalized or not?

Reply:
Alright, so I went back through this, and there's basically three utilization's here:

Primary Weft. This is a proper noun, a named specific thing, what the Remnant call the dimension that we live in.
Secondary Weft. This is also a proper noun, a specific thing, what the Remnant call the dimension that the Shattered Light is partially in.

weft. This is a general noun. Not actually synonymous with "dimension" but reasonably similar. So if they're talking about the weft, then they're talking about it in a general sense; For instance, they consider generating a weft field to manipulate how it interacts with the Primary and Secondary Wefts, sort of like a warp field, but obviously different functionalities and effects.

It is possible that there are instances of The Weft that I have missed, which would shortened forms for whatever weft the speaker is focusing on, but probably the Primary Weft. 

(so weft vs Weft is kind of a "There's lots of trees, but that one is The Tree that I care about." sort of thing.)